.github/workflows/stage.yml

```diff
@@ -10,25 +10,70 @@ on:
 jobs:
   release_pr:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [ release-chrome, release-firefox ]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js 16
-      uses: actions/setup-node@v1
-      with:
-        node-version: 16.x
-    - name: Install dependencies
-      run: |
-        npm i -g web-ext
-        npm run install-ci
-    - name: Fetch config and update version
-      run: |
-        npm run bundle-config
-        node scripts/bumpVersion.js ${{ github.event.inputs.version }}
-    - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v3
-      with:
-        commit-message: ${{ github.event.inputs.version }}
-        branch: "release-${{ github.event.inputs.version }}"
-        title: "Prepare release ${{ github.event.inputs.version }}"
-        body: ''
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: |
+          npm i -g web-ext
+          npm run install-ci
+      - name: Fetch config and update version
+        run: |
+          npm run bundle-config
+          node scripts/bumpVersion.js ${{ github.event.inputs.version }}
+      - name: Commit config and version updates
+        uses: stefanzweifel/git-auto-commit-action@8c3ed373fb7e451dbb9706bf99ad63124e33f614
+        with:
+          commit_message: ${{ github.event.inputs.version }}
+          branch: develop
+          title: "Prepare release ${{ github.event.inputs.version }}"
+
+      - name: Generate changelog
+        id: changelog
+        uses: metcalfc/changelog-generator@c3f61b2a3db543ceee83c759ac8e48419a1a2211
+        with:
+          myToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

Comment:
This should be moved to before the `install-ci` step (as in https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/.github/workflows/tests.yml#L14-L20). This is step restores a cached `.npm` folder for so the `npm install` artifacts don't need to be fetched.

See https://github.com/actions/cache/blob/main/examples.md#node---npm for more on how this cache step works.

Reply:
Thanks for noticing. I've pushed the change. Do you mind checking again if I got it right this time 🙂?